### ISSUE TITLE
Introducing AI-assisted reading features.

### ISSUE THREAD

**Issue by bestZwei** [Athou/commafeed#1639]
Now that we are in the era of AI, I hope we can consider adding a feature: utilizing AI interfaces for translation, summarization, or dialogue related to article content. Initially, I believe compatibility with the OpenAI format would suffice, as the interfaces of most model providers can be converted to a format compatible with the OpenAI interface through services like NewAPI or OneAPI. Some other RSS readers have already made open attempts in this area, such as Huntly and a certain extension for FreshRSS.

**Comment by dstutz** [comment]
I could not disagree more.  I do not want ML-related (it's not AI) features added to CommaFeed, but that's just my opinion which happens to be polar opposite of your opinion.  There's an API, feel free to use it to access the feeds and perform all the external processing you want.

**Comment by Athou** [comment]
I tend to agree with @dstutz, I try to keep CommaFeed lean and simple. You could probably implement this in a standalone browser extension though, it might even already exist.
